tests/src/baseservices/threading/interlocked/compareexchange/compareexchangetstring.cs

```diff
@@ -95,8 +95,9 @@ private string AddToTotal(string addend)
             initialValue = Val;
             newValue = initialValue + addend;
         } 
-        while (initialValue != Interlocked.CompareExchange<string>(
```

Comment:
Strings here get pretty long and text compare is unnecessary expensive, not just makes the test slow, but partially defeats the purpose of the tests since it spends most of the time comparing strings.